src/main-refactor/network/10-vpc.tf

```
@@ -0,0 +1,89 @@
+locals {
+  eks_workload_cidrs      = ["10.0.0.0/21", "10.0.8.0/21", "10.0.16.0/21"]
+  eks_control_plane_cidrs = ["10.0.24.0/24", "10.0.25.0/24", "10.0.26.0/24"]
+  probing_lb_cidrs        = ["10.0.27.0/24", "10.0.28.0/24", "10.0.29.0/24"]
```

I forgot to add subnets for Load balancers.

---

```suggestion
  int_lbs_cidrs        = ["10.0.27.0/24", "10.0.28.0/24", "10.0.29.0/24"]
```
Since we need to use APIGW, these load balancer will be internal